src/components/app/Home.js

```diff
@@ -267,7 +267,23 @@ class Home extends React.PureComponent<HomeProps, HomeState> {
       return;
     }
 
-    const { files } = event.dataTransfer;
+    const {
+      files,
+      items: [firstItem],
+    } = event.dataTransfer;
+
+    if ('webkitGetAsEntry' in firstItem) {
```

Comment:
[MDN says](https://developer.mozilla.org/en-US/docs/Web/API/DataTransferItem/webkitGetAsEntry):

> This function is implemented as webkitGetAsEntry() in non-WebKit browsers including Firefox at this time; it may be renamed to simply getAsEntry() in the future, so you should code defensively, looking for both.

Maybe we should do it?
Like:
```js
let entry = null;
if ('webkitGetAsEntry' in firstItem) {
  webkitEntry = firstItem.webkitGetAsEntry();
} else if ('getAsEntry' in firstItem) {
  webkitEntry = firstItem.getAsEntry();
}

if (entry !== null) {
  // rest of code
}
```

What do you think?

Reply:
Yes, I will make the change accordingly. Thanks for the suggestion :)

Reply:
@julienw I tried for the same. But as the `getAsEntry` is also a non-standard function. I will need to add another `$FlowFixMe`. Will it be fine with you?
cc: @canaltinova

Reply:
Yeah, Flow support and Browser support are 2 different things :)

Reply:
@julienw Made the changes.
Also filed an issue #2230.
cc: @canaltinova